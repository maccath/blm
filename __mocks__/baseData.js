const baseData = {
  site: {
    siteMetadata: {
      author: "Site Author",
      description: "Site description",
      title: "Site Title",
    },
  },
  placeholderImage: {
    childImageSharp: {
      fluid: {
        src: "icon.png",
        srcSet: "icon.png 50w, icon.png 100w",
        sizes: "(max-width: 200px) 100vw, 200px",
        base64: "data:image/png;base64,iVBORw0KG",
        aspectRatio: 1,
      },
    },
  },
}

export default baseData
