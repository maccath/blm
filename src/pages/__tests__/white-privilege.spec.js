import React from "react"
import renderer from "react-test-renderer"
import { useStaticQuery } from "gatsby"
import "jest-styled-components"
import baseData from "../../../__mocks__/baseData"

import WhitePrivilege from "../white-privilege"
import Layout from "../../components/layout"

beforeEach(() => {
  useStaticQuery.mockImplementation(() => baseData)
})

describe("White Privilege page", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Layout>
          <WhitePrivilege />
        </Layout>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
