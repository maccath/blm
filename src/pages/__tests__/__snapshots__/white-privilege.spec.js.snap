// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`White Privilege page renders correctly 1`] = `
<div
  style={
    Object {
      "margin": "0 auto",
      "maxWidth": 960,
      "padding": "0 1.0875rem 1.45rem",
    }
  }
>
  <main>
    <h1>
      White privilege?
    </h1>
    <ul>
      <li>
        <a
          href="https://everydayfeminism.com/2014/09/white-privilege-explained/white-privilege.mdx"
        >
          White Privilege Explained - in Comic Form
        </a>
      </li>
      <li>
        <a
          href="http://www.raceconscious.org/2017/07/race-matters-story-white-privilege"
        >
          Race Matters - A Story About White Privilege
        </a>
      </li>
      <li>
        <a
          href="https://www.tolerance.org/magazine/fall-2018/what-is-white-privilege-really"
        >
          What is White Privilege?
        </a>
      </li>
      <li>
        <a
          href="https://goodmenproject.com/featured-content/white-fragility-why-its-so-hard-to-talk-to-white-people-about-racism-twlm/"
        >
          Why it's so hard to talk to white people about racism
        </a>
      </li>
      <li>
        <a
          href="https://www.theatlantic.com/politics/archive/2017/10/the-language-of-white-supremacy/542148/"
        >
          The Language of White Supremacy
        </a>
      </li>
    </ul>
    <div
      className="mt-6 flex space-x-4"
    >
      <a
        class="flex-initial rounded-lg bg-white text-purple p-3 no-underline"
        href="/why-care"
      >
        Ok, next.
      </a>
    </div>
  </main>
  <footer
    className="css-1bxtzlt"
  >
    <p>
      Compiled by 
      <a
        href="http://katy-ereira.co.uk"
      >
        Katy Ereira
      </a>
       /
       
      <a
        href="http://twitter.com/maccath"
      >
        @maccath
      </a>
       | Fist icon by
       
      <a
        href="https://thenounproject.com/cesarreynoso562/"
      >
        Cesar Reynoso
      </a>
    </p>
  </footer>
</div>
`;
